addon/window/browser.js

```diff
@@ -0,0 +1,35 @@
+import Base from './base';
```

Comment:
Would work for sure. Just did not want to not duplicated code, but will revisit this...